src/main/java/core/basesyntax/serviceimpl/CsvFileReaderServiceImpl.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.CsvFileReaderService;
+import core.basesyntax.strategy.TransactionsStrategy;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+public class CsvFileReaderServiceImpl implements CsvFileReaderService {

```

Comment:
According to the name, this class should only read csv files and return a list of strings. then you should process this data in another class. that's the single responsibility principle.